sherpa/astro/io/dummy_backend.py

```diff
@@ -19,42 +19,766 @@
 #
 '''A dummy backend for I/O.
 
-This backend provides no functionality and raises an error if any of its
-functions are used. It is just here to ensure that `sherpa.astro.io` can be
-imported, even if no FITS reader is installed.
+This backend provides no functionality and raises an error if any of
+its functions are used. It is provided as a model for what routines
+are needed in a backend, even if it does nothing, and to allow
+`sherpa.astro.io` to be imported even if no usable backend is
+available.
+
 '''
+
 import logging
+from typing import Any, Optional, Sequence
+
+import numpy as np
+
+from ..data import Data1D
+from .types import NamesType, HdrTypeArg, HdrType, \
+    ColumnsType, ColumnsTypeArg, DataTypeArg, DataType
+from .xstable import TableHDU
 
 __all__ = ('get_table_data', 'get_header_data', 'get_image_data',
-           'get_column_data', 'get_ascii_data',
-           'get_arf_data', 'get_rmf_data', 'get_pha_data',
+           'get_column_data', 'get_ascii_data', 'get_arf_data',
+           'get_rmf_data', 'get_pha_data',
+           #
```

Comment:
Is there some significance of adding the comment lines before and after the pack_* methods?

Reply:
There's no reason for this Python wise - it was just useful for me to see them separate from the other routines (given the number of times I've tried to get this done). I can remove them if needed.